### DISCUSSION TITLE
Support for Panel 1.0rc?

### DISCUSSION THREAD

**Discussion by nascif** [pyscript/pyscript#1461] [Q&A]
Hello! I am trying to port a PyScript app to [Panel 1.0](https://community.anaconda.cloud/t/panel-1-0-0-release-candidate-is-out-with-many-exciting-new-features/54766), but I can’t find a suitable Bokeh version to go with it. The migration guide said that it depends on Bokeh 3.x. 3.1.0 doesn’t seem to be available yet, and 3.0.3 doesn’t meet the versioning criteria.

Has anybody else got the latest panel to run on PyScript?
Thanks!

**Comment by JeffersGlass** [comment]
We’re currently working on upgrading PyScript to use Pyodide 0.23.x for the next release, which pins Bokeh 3.1.0. Progress on that process is tracked in #1347 - don’t think there are major blockers, just a matter of getting our test setup to behave.

**Comment by philippjfr** [comment]
For the time being (or even beyond then) you can use our custom wheels which are uploaded with every release:

https://cdn.holoviz.org/panel/1.0.0-rc.8/dist/wheels/bokeh-3.1.1-py3-none-any.whl
https://cdn.holoviz.org/panel/1.0.0-rc.8/dist/wheels/panel-1.0.0rc8-py3-none-any.whl

We build these because Bokeh and Panel bundle a bunch of resources that aren't needed in a pyodide/pyscript context and the download size is reduced >10x this way.
